presto-main/src/main/java/io/prestosql/server/remotetask/HttpRemoteTask.java

```diff
@@ -509,7 +516,11 @@ private synchronized void sendUpdate()
                 fragment,
                 sources,
                 outputBuffers.get(),
-                totalPartitions);
+                totalPartitions,
+                dynamicFilters.entrySet().stream()
```

Comment:
In this implementation we send dynamic filters only along with scheduling of splits. Ideally we should send dynamic filters as they become ready. Current approach works reasonably well for hive as it generates splits in batches.